### PR TITLE
ign -> gz : Release Tools Migrate Links and Lib Names

### PR DESCRIPTION
**Pull request by methylDragon**
Part of: https://github.com/gazebo-tooling/release-tools/pull/746

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/release_collection.md [hunk 1]
```diff
@@ -67,15 +67,15 @@ When opening PRs, add a link back to this issue for easier tracking.
...
+    - [ ] Source code ([example](https://github.com/gazebosim/gz-common/pull/193))
         * <!-- LINK PRs HERE -->
     - [ ] Add files to `gazebodistro` ([example](https://github.com/gazebo-tooling/gazebodistro/pull/12))
         * <!-- LINK PR HERE -->
     - [ ] Add aliases to `homebrew-simulation` ([example](https://github.com/osrf/homebrew-simulation/commit/1f8602af6f52e06e0542eebfbdbe97f5f6cf950c))
         * <!-- LINK PR HERE -->
     - [ ] Create new `-release` repositories (use [this script](https://github.com/gazebo-tooling/release-tools/blob/master/release-repo-scripts/new_ignition_release_repos.bash))
     - [ ] Enable nightlies for all `main` branches on `gzdev` ([example](https://github.com/gazebo-tooling/gzdev/pull/50))
-    - [ ] Execute the tick-tock's "tock" for deprecations ([example](https://github.com/ignitionrobotics/ign-gazebo/pull/875))
+    - [ ] Execute the tick-tock's "tock" for deprecations ([example](https://github.com/gazebosim/gz-gazebo/pull/875))
```
Comment (chapulina) [inline .github/ISSUE_TEMPLATE/release_collection.md:78]:
```suggestion
    - [ ] Execute the tick-tock's "tock" for deprecations ([example](https://github.com/gazebosim/gz-sim/pull/875))
```

File: .github/ISSUE_TEMPLATE/release_collection.md [hunk 2]
```diff
@@ -90,24 +90,24 @@ If the collection will be officially paired with a ROS 2 distro:
...
-[ign-gazebo](https://github.com/ignitionrobotics/ign-gazebo)         |  | ign-gazeboN | ✔️ | ✔️ | ✔️ | ✔️ | ✔️ | [PRs](https://github.com/ignitionrobotics/ign-gazebo/pulls/?q=is%3Apr+is%3Aopen+label%3A%22beta%22) |
-[ign-launch](https://github.com/ignitionrobotics/ign-launch)         |  | ign-launchN | ✔️ | ✔️ | ✔️ | ✔️ | ✔️ | [PRs](https://github.com/ignitionrobotics/ign-launch/pulls/?q=is%3Apr+is%3Aopen+label%3A%22beta%22) |
-[ign-<collection>](https://github.com/ignitionrobotics/ign-<cok>)    |  | main | ✔️ | ✔️  | ✔️ | ✔️ | ✔️ | [PRs](https://github.com/ignitionrobotics/ign-<collection>/pulls/?q=is%3Apr+is%3Aopen+label%3A%22beta%22) |
-[ros_ign](https://github.com/ignitionrobotics/ros_ign)               |  | <distros> | ✔️ | ✔️ | ✔️ |  |  | [PRs](https://github.com/ignitionrobotics/ros_ign/pulls/?q=is%3Apr+is%3Aopen+label%3A%22beta%22) |
+[ign-tools](https://github.com/gazebosim/gz-tools)           |  | ign-toolsN | ✔️ | ✔️  | ✔️ | ✔️ | ✔️ | [PRs](https://github.com/gazebosim/gz-tools/pulls/?q=is%3Apr+is%3Aopen+label%3A%22beta%22) |
```
Comment (chapulina) [inline .github/ISSUE_TEMPLATE/release_collection.md:93]:
```suggestion
[gz-tools](https://github.com/gazebosim/gz-tools)           |  | ign-toolsN | ✔️ | ✔️  | ✔️ | ✔️ | ✔️ | [PRs](https://github.com/gazebosim/gz-tools/pulls/?q=is%3Apr+is%3Aopen+label%3A%22beta%22) |
```

File: .github/ISSUE_TEMPLATE/release_collection.md [hunk 3]
```diff
@@ -90,24 +90,24 @@ If the collection will be officially paired with a ROS 2 distro:
...
+[ign-transport](https://github.com/gazebosim/gz-transport)   |  | ign-transportN | ✔️ | ✔️ | ✔️ | ✔️ | ✔️ | [PRs](https://github.com/gazebosim/gz-transport/pulls/?q=is%3Apr+is%3Aopen+label%3A%22beta%22) |
+[ign-gui](https://github.com/gazebosim/gz-gui)               |  | ign-guiN | ✔️ | ✔️  | ✔️ | ✔️ | ✔️ | [PRs](https://github.com/gazebosim/gz-gui/pulls/?q=is%3Apr+is%3Aopen+label%3A%22beta%22) |
+[ign-sensors](https://github.com/gazebosim/gz-sensors)       |  | ign-sensorsN | ✔️ | ✔️ | ✔️ | ✔️ | ✔️ | [PRs](https://github.com/gazebosim/gz-sensors/pulls/?q=is%3Apr+is%3Aopen+label%3A%22beta%22) |
+[ign-physics](https://github.com/gazebosim/gz-physics)       |  | ign-physicsN | ✔️ | ✔️ | ✔️ | ✔️ | ✔️ | [PRs](https://github.com/gazebosim/gz-physics/pulls/?q=is%3Apr+is%3Aopen+label%3A%22beta%22) |
+[ign-gazebo](https://github.com/gazebosim/gz-gazebo)         |  | ign-gazeboN | ✔️ | ✔️ | ✔️ | ✔️ | ✔️ | [PRs](https://github.com/gazebosim/gz-gazebo/pulls/?q=is%3Apr+is%3Aopen+label%3A%22beta%22) |
```
Comment (chapulina) [inline .github/ISSUE_TEMPLATE/release_collection.md:107]:
```suggestion
[gz-sim](https://github.com/gazebosim/gz-sim)         |  | gz-simN | ✔️ | ✔️ | ✔️ | ✔️ | ✔️ | [PRs](https://github.com/gazebosim/gz-gazebo/pulls/?q=is%3Apr+is%3Aopen+label%3A%22beta%22) |
```

File: .github/ISSUE_TEMPLATE/release_collection.md [hunk 4]
```diff
@@ -90,24 +90,24 @@ If the collection will be officially paired with a ROS 2 distro:
...
+[ign-launch](https://github.com/gazebosim/gz-launch)         |  | ign-launchN | ✔️ | ✔️ | ✔️ | ✔️ | ✔️ | [PRs](https://github.com/gazebosim/gz-launch/pulls/?q=is%3Apr+is%3Aopen+label%3A%22beta%22) |
+[ign-<collection>](https://github.com/gazebosim/gz-<cok>)    |  | main | ✔️ | ✔️  | ✔️ | ✔️ | ✔️ | [PRs](https://github.com/gazebosim/gz-<collection>/pulls/?q=is%3Apr+is%3Aopen+label%3A%22beta%22) |
+[ros_ign](https://github.com/gazebosim/ros_ign)               |  | <distros> | ✔️ | ✔️ | ✔️ |  |  | [PRs](https://github.com/gazebosim/ros_ign/pulls/?q=is%3Apr+is%3Aopen+label%3A%22beta%22) |
```
Comment (chapulina) [inline .github/ISSUE_TEMPLATE/release_collection.md:110]:
```suggestion
[ros_gz](https://github.com/gazebosim/ros_gz)               |  | <distros> | ✔️ | ✔️ | ✔️ |  |  | [PRs](https://github.com/gazebosim/ros_gz/pulls/?q=is%3Apr+is%3Aopen+label%3A%22beta%22) |
```

File: README.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 # Release Tools
 
 This repository holds scripts and tools that are used for testing and releasing
-Gazebo Classic and Ignition software.
+Gazebo Classic and Gazebo software.
```
Comment (chapulina) [inline README.md:4]:
```suggestion
Gazebo software.
```

File: README.md [hunk 6]
```diff
@@ -33,10 +33,10 @@ Gazebo Classic and Ignition software.
     * Bump minor if there are new backwards-compatible features, like new APIs.
     * Bump patch if there are only bug fixes that won't affect the API.
 1. Open pull request updating the version on the source code
-    * [Example pull request.](https://github.com/ignitionrobotics/ign-physics/pull/132)
+    * [Example pull request.](https://github.com/gazebosim/gz-physics/pull/132)
     * Update the version on `CMakeLists.txt`.
     * Include a link comparing the current branch to the latest release.
-        1. [Example branch comparison.](https://github.com/ignitionrobotics/ign-gazebo/compare/ignition-gazebo3_3.5.0...ign-gazebo3)
+        1. [Example branch comparison.](https://github.com/gazebosim/gz-gazebo/compare/ignition-gazebo3_3.5.0...ign-gazebo3)
```
Comment (chapulina) [inline README.md:39]:
```suggestion
        1. [Example branch comparison.](https://github.com/gazebosim/gz-sim/compare/ignition-gazebo3_3.5.0...ign-gazebo3)
```

File: README.md [hunk 7]
```diff
@@ -46,10 +46,10 @@ Gazebo Classic and Ignition software.
         1. Verify the Changelog.md entries using the GitHub branch comparison.
     * Update the migration guide as needed.
     * Wait for this pull request to be merged before proceeding.
-1. Clone the appropriate release repository from https://github.com/ignition-release.
-    * For example, `git clone https://github.com/ignition-release/ign-physics3-release`
+1. Clone the appropriate release repository from https://github.com/gazebo-release.
+    * For example, `git clone https://github.com/gazebo-release/ign-physics3-release`
```
Comment (chapulina) [inline README.md:50]:
```suggestion
    * For example, `git clone https://github.com/gazebo-release/gz-physics3-release`
```

File: README.md [hunk 8]
```diff
@@ -71,20 +71,20 @@ Gazebo Classic and Ignition software.
...
+        1. If you observe a failure of an Gazebo Dome package, check the list of supported
            architectures in [issue #297](https://github.com/gazebo-tooling/release-tools/issues/297);
            for example Bionic/armhf and Focal/armhf are not supported for Dome.
            To disable builds for a specific architecture, add a file to the `-release` repository that starts with `.releasepy_NO_ARCH_`
            and append the name of the architecture to be excluded. The file can be added to the
-           root of the `-release` repository (like [.releasepy_NO_ARCH_ARM64 in ign-gazebo2-release](https://github.com/ignition-release/ign-gazebo2-release/blob/master/.releasepy_NO_ARCH_ARM64))
-           or in a distro-specific sub-folder (like [bionic/.releasepy_NO_ARCH_armhf in ign-launch2-release](https://github.com/ignition-release/ign-launch2-release/blob/master/bionic/.releasepy_NO_ARCH_armhf)).
+           root of the `-release` repository (like [.releasepy_NO_ARCH_ARM64 in ign-gazebo2-release](https://github.com/gazebo-release/ign-gazebo2-release/blob/master/.releasepy_NO_ARCH_ARM64))
```
Comment (chapulina) [inline README.md:79]:
```suggestion
           root of the `-release` repository (like [.releasepy_NO_ARCH_ARM64 in gz-gazebo2-release](https://github.com/gazebo-release/gz-gazebo2-release/blob/master/.releasepy_NO_ARCH_ARM64))
```

File: jenkins-scripts/docker/lib/_generic_linux_compilation_build.sh.bash [hunk 9]
```diff
@@ -55,7 +55,7 @@ cat >> build.sh << DELIM_BUILD_DEPS
       dependency_repo="osrf/${dep}"
     else
       # need to replace _ by -
-      dependency_repo="ignitionrobotics/${dep//_/-}"
+      dependency_repo="gazebosim/${dep//_/-}"
```
Comment (chapulina) [inline jenkins-scripts/docker/lib/_generic_linux_compilation_build.sh.bash:58]: Did you track down how we end up here? It's possible that there's `ignitionrobotics` hardcoded somewhere else still

File: jenkins-scripts/docker/lib/gazebo-base-default.bash [hunk 10]
```diff
@@ -134,7 +134,7 @@ cat >> build.sh << DELIM_BUILD_DEPS
       dependency_repo="osrf/${dep}"
     else
       # need to replace _ by -
-      dependency_repo="ignitionrobotics/${dep/_/-}"
+      dependency_repo="gazebosim/${dep/_/-}"
```
Comment (chapulina) [inline jenkins-scripts/docker/lib/gazebo-base-default.bash:137]: Here too

File: README.md (hunk 8 above)
Comment (methylDragon) [reply README.md:79]: Older releases don't have a `gz` version :X

Both these links are broken
https://github.com/gazebo-release/gz-sim2-release
https://github.com/gazebo-release/gz-gazebo2-release

File: README.md (hunk 7 above)
Comment (methylDragon) [reply README.md:50]: https://github.com/gazebo-release/gz-physics3-release doesn't exist :X

File: jenkins-scripts/docker/lib/_generic_linux_compilation_build.sh.bash (hunk 9 above)
Comment (methylDragon) [reply jenkins-scripts/docker/lib/_generic_linux_compilation_build.sh.bash:58]: image

It's just used for github cloning! Should be safe

File: jenkins-scripts/docker/lib/gazebo-base-default.bash (hunk 10 above)
Comment (methylDragon) [reply jenkins-scripts/docker/lib/gazebo-base-default.bash:137]: image

It's just used for github cloning! Should be safe

File: source-repo-scripts/release_summary.bash [hunk 11]
```diff
@@ -38,7 +38,7 @@ echo "NAME_FOR_TAGS: $NAME_FOR_TAGS"
 NAME_FOR_REPO=gz-${LIB}
 NAME_FOR_REPO="${NAME_FOR_REPO/_/-}"
 NAME_FOR_REPO="${NAME_FOR_REPO/gz-sdformat/sdformat}"
-NAME_FOR_REPO="${NAME_FOR_REPO/gz-gazebo/gz-sim}"
+NAME_FOR_REPO="${NAME_FOR_REPO/gz-sim/gz-sim}"
```
Comment (chapulina) [inline source-repo-scripts/release_summary.bash:41]: 🧐

**Review by chapulina — APPROVED** [review]
LGTM, just one detail left

**Pull request merged**